portal-api/src/v1/api.js

```diff
@@ -4,6 +4,7 @@ require('dotenv').config();
 
 const centralApiUrl = process.env.DTFS_CENTRAL_API;
 const tfmUrl = process.env.TFM_API;
+const { API_KEY } = process.env;
 
```

Comment:
Can we please create a global const named `headers` with the defined values, in order to avoid code repetition.

Reply:
done